controls/SV-221588.rb

```diff
@@ -56,4 +56,22 @@
   tag legacy: ["SV-94635","V-79931"]
   tag cci: ["CCI-000169"]
   tag nist: ["AU-12 a"]
+
+  if input('siprnet_system') == true
```

Comment:
The 'sensitive_system' input should be a boolean so you should not need to compair it to a string true or false.

Also, let's avoid using 'nipr' sipr etc - let's use generic terms like 'sensitive' etc so that we keep as many folks enguaged as possible.

Reply:
Like this   **if sensitive_system == true**   ?